apps/epic-react/src/components/app/navigation/index.tsx

```diff
@@ -169,7 +151,7 @@ const Navigation: React.FC<NavigationProps> = ({navChildren}) => {
                 ),
               )}
             >
-              Workshops
+              Modules
             </Link>
```

Comment:
ah, I didn't see this link because of the `sellingLive` flag (I believe)

Reply:
Yeah